cmd/bosun/sched/sched.go

```diff
@@ -22,6 +22,8 @@ import (
 	"github.com/MiniProfiler/go/miniprofiler"
 	"github.com/boltdb/bolt"
 	"github.com/bradfitz/slice"
+	"github.com/kylebrandt/boolq"
+	"github.com/kylebrandt/boolq/parse"
```

Comment:
can we alias this reference please. We have way too many things called parse. Better yet, maybe it should just be `boolq.Parse` and nobody else needs to know about the internal parse package.